master-firmware/src/base/base_controller.c

```diff
@@ -39,14 +38,21 @@ void robot_init(void)
 
     rs_set_left_pwm(&robot.rs, cvra_motor_left_wheel_set_velocity, &left_wheel_motor);
     rs_set_right_pwm(&robot.rs, cvra_motor_right_wheel_set_velocity, &right_wheel_motor);
-    rs_set_left_ext_encoder(&robot.rs, cvra_encoder_get_left_ext, NULL, LEFT_WHEEL_CORRECTION_FACTOR);
-    rs_set_right_ext_encoder(&robot.rs, cvra_encoder_get_right_ext, NULL, RIGHT_WHEEL_CORRECTION_FACTOR);
+    rs_set_left_ext_encoder(
+        &robot.rs, cvra_encoder_get_left_ext, NULL,
+        config_get_scalar("master/odometry/left_wheel_correction_factor"));
+    rs_set_right_ext_encoder(
+        &robot.rs, cvra_encoder_get_right_ext, NULL,
+        config_get_scalar("master/odometry/right_wheel_correction_factor"));
 
     /* Position manager */
     position_init(&robot.pos);
     position_set_related_robot_system(&robot.pos, &robot.rs); // Link pos manager to robot system
 
-    position_set_physical_params(&robot.pos, ROBOT_EXTERNAL_TRACK_LENGTH_MM, EXTERNAL_ENCODER_TICKS_PER_MM);
+    position_set_physical_params(
```

Comment:
There is one in cvra/coding style. But I agree that it may not be the best place.